### PR TITLE
adding priority class

### PR DESCRIPTION
**Pull request by Jennls82**
adding a priority class config to the chart 

### Checklist
- [x] Chart version bumped

### CODE REVIEW

File: chart/tugger/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| slackDedupeTTL | string | `nil` |  |
+| tls.caCert | string | `nil` |  |
+| tls.secretName | string | `nil` |  |
+| tolerations | list | `[]` |  |
+| webhookUrl | string | `nil` |  |
+| whitelistNamespaces[0] | string | `"kube-system"` |  |
+| whitelistRegistries | list | `[]` |  |
+
+----------------------------------------------
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```
Comment (kd7lxl) [inline chart/tugger/README.md:69]: I'm not in favor of committing generated code without some automation to generate it. It's just going to become stale upon the next change. This type of change belongs in its own PR where the merits can be discussed without blocking your other change.

**Review by kd7lxl — CHANGES_REQUESTED** [review]
`priorityClassName` change LGTM.

File: chart/tugger/README.md (hunk 1 above)
Comment (Jennls82) [reply chart/tugger/README.md]: sgtm removed in next commit

**Review by kd7lxl — APPROVED** [review]

**Review by jainishshah17 — APPROVED** [review]
LGTM

**Pull request merged**